setup.py

```diff
@@ -1,7 +1,26 @@
 #!/usr/bin/env python
+import os
+import re
+import codecs
+
 from setuptools import setup
 
 
+def read_file(filename, encoding='utf8'):
+    """read unicode from given file"""
```

Comment:
It doesn't matter that much here, but `__doc__` should start with capital letter, as in sentence. It looks nicer in docs. :)